src/lib/y2users/password.rb

```diff
@@ -57,6 +57,16 @@ class Password
     # @return [:local, :ldap, :unknown] where is user defined
     attr_reader :source
 
+    def self.plain(value)
```

Comment:
NP: Maybe *new_plain* would be more auto-explanatory?

Reply:
well, I do not like much that new prefix. I more prefer `create_plain` or `build_plain`, but both is quite long. But if you prefer more auto explanatory, it is fine for me.